developer/architecture/metadata.md

```diff
@@ -1,11 +1,11 @@
 # SEO and Metadata
 
-Basic meta tag data is inserted by the `Router.DocHead.init` method which uses [kadira:dochead](https://github.com/kadirahq/meteor-dochead) package to manipulate `document.head`.
+Basic meta tag data is inserted by the `Router.DocHead.init` method which uses the `dom` core package to manipulate `document.head`.
```

Comment:
I don't think this is correct. Will doublecheck, but there is no `Router.DocHead.init` in the project that I can find.